include/grpc/grpc_security.h

```diff
@@ -320,6 +355,36 @@ GRPCAPI grpc_server_credentials *grpc_ssl_server_credentials_create_ex(
     grpc_ssl_client_certificate_request_type client_certificate_request,
     void *reserved);
 
+typedef struct grpc_ssl_server_credentials_options
+    grpc_ssl_server_credentials_options;
+
+/** Creates an options object using a certificate config. Use this method when
+   the certificates and keys of the SSL server will not changes during the
+   server's lifetime. */
```

Comment:
we should mention that this function makes a copy of argument config. and because of that, i think the argument should be `const grpc_ssl_server_certificate_config *certificate_config`; that also helps make it clear.

Reply:
Great suggestion. Will fix.